src/AVFoundation/AVAudioUnitComponentManager.cs

```diff
@@ -0,0 +1,35 @@
+//
+// AVAudioUnitComponentManager.cs
+//
+// Author: Vincent Dondain (vidondai@microsoft.com)
+//
+// Copyright 2018 Microsoft.
+//
+
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.AVFoundation {
+
+#if !WATCH
+	public partial class AVAudioUnitComponentManager {
+#if !XAMCORE_4_0
+		[Obsolete ("Please use the static 'SharedInstance' property as this type is not meant to be created.")]
+		[CompilerGenerated]
+		[EditorBrowsable (EditorBrowsableState.Never)]
+		[Export ("init")]
+		public AVAudioUnitComponentManager () : base (NSObjectFlag.Empty)
```

Comment:
I think you can inline this directly in the ApiDefinition so no need for an additional manual code file 

```
[Obsolete ("Please use the static 'SharedInstance' property as this type is not meant to be created.")]
[Export ("init")]
IntPtr Constructor ();
```